README.md

```diff
@@ -39,38 +38,97 @@ print(favoriteSentence)
 
 ```swift
-print("It's a UNIX system! I know this!")
-``` 
-* Challenge the student at this point to print the following sentence. "Hi Mom! My favorite word is serendipity, my favorite color is green, and my favorite singer is Billy Joel" 
-* After the prints that sentence (not knowing yet about string interpolation), pose the quesiton.. "Wouldn't it be easier if we could print the values of various variables within that sentence we structured in the print function"? Or something like that.. then going into string interpolation.
+print(favoriteWord)
+print(favoriteColor)
+print(favoriteMusician)
+```
+
+Now you will see your program's output in the bottom pane of the playground:
+
+![print() output](.images/xcode3.png)
+
+## Printing Strings
+
+Swift's `print()` function can be used to print anything, not just variables and constants. You can also print entire strings. Just like variables and constants, you print a string by passing it to the `print()` function. Put this in your playground file and see what happens:
 
 ```swift
-print("Hi Mom! My favorite word is \(favoriteWord), my favorite color is \(favoriteColor), and my favorite singer is \(favoriteSinger).")
+print("I'm printing a string in Swift!")
 ```
-* Have the student get used to the syntax, asking them to print out a few things in the playground file.
-* Challenge the student to be able to create something like this: 
+
+You should see the string "I'm printing a string in Swift!" in the playground's output pane, along with the other variables you printed before.
+
+You can even print longer sentences using the `print()` function. There's no limit to what you can print! Try printing the sentence "Hi Mom! My favorite word is serendipity, my favorite color is green, and my favorite musician is Billy Joel." and see what happens:
 
 ```swift
-let favoriteSentence = "Hello everyone, how are you doing? My favorite word is \(favoriteWord)"
-print(favoriteSentence)
+print("Hi Mom! My favorite word is serendipity, my favorite color is green, and my favorite musician is Billy Joel.")
 ```
 
-* Then show them how (if they can't figure it out)
+## Combining Strings and Variables
+
+At this point you may be thinking, "Wait a minute! Serependity doesn't exactly roll off the tongue, I hate green, and Billy Joel makes my ears bleed!" You've probably also remembered that you already declared variables to hold your favorite word, color, and musician, so why are you having to pledge your undying love for these things you don't like? Why can't you print out your own preferences?
+
+Guess what? You can! Swift has a very helpful feature to do combine strings and variables: _interpolation_. When you interpolate a string, you can mix in a variable (or constant) you declared before with a string. Swift will calculate a new string based on the one you provided and the variables or constants you wish to be interpolated.
+
```

Comment:
> Swift has a very helpful feature to do combine strings and variables: _interpolation_.

Change this to say something like this instead... "Swift has a very helpful feature to combining strings and variables: _interpolation_."

> When you interpolate a string, you can mix in a variable (or constant) you declared before with a string.

Maybe say something like this.. "When you interpolate a string, you can mix in a variable (or constant) you declared before along with a string." 

> Swift will calculate a new string based on the one you provided and the variables or constants you wish to be interpolated.

Instead of stating "... one you provided **and** the variables or constants.." we can swap out the word and with **along with**. What do you think?